lib/node_modules/@stdlib/math/base/special/cosc/README.md

```diff
@@ -0,0 +1,102 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2018 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# cosc
+
+> Compute the derivative of the [cardinal sine][sinc] of a number.
+
+<section class="intro">
+
+The derivative of the normalized [cardinal sine][sinc] function is defined as
+
+` { cosc(x) = (cos( PI*x ) - sin( PI*x )/(PI*x)) / x } if x != 0, and 0 if x == 0 `
```

Comment:
This should be formatted as a TeX equation. We have a specific way of doing this. For an example, see https://raw.githubusercontent.com/stdlib-js/stdlib/develop/lib/node_modules/%40stdlib/math/base/special/acovercos/README.md.

So for this package, we would do

```
<!-- <equation class="equation" label="eq:cosc" align="center" raw=""\operatorname{cosc}(x) = \begin{cases} \frac{\cos(\pi x)}{x} - \frac{\sin( \pi x)}{\pi x^2} & \textrm{if}\ x \neq 0 \\ 0 & \textrm{if}\ x = 0\end{cases}" alt="Derivative of the normalized cardinal sine."> -->

<div class="equation" align="center" data-raw-text="\operatorname{cosc}(x) = \begin{cases} \frac{\cos(\pi x)}{x} - \frac{\sin( \pi x)}{\pi x^2} &amp; \textrm{if}\ x \neq 0 \\ 0 &amp; \textrm{if}\ x = 0\end{cases}" data-equation="eq:cosc">
    <img src="" alt="Derivative of the normalized cardinal sine.">
    <br>
</div>

<!-- </equation> -->
```

If you ever need to quickly preview how a TeX equation will render, you can use [codecogs](https://www.codecogs.com/eqnedit.php).

Reply:
Fixed, and added an SVG of the equation.